### PR TITLE
Improve duplicates display and detection

### PR DESCRIPTION
**Pull request by ZeHiro**
I made two different commits as they are two different improvements.
The new defined treshold is not modifiable through the UI. I can add this if necessary.
See #212 for explanations of second point.

### CODE REVIEW

**Comment by bnjbvr** [comment]
Thanks for the PR. I actually had a case once where I've reloaded operations several times during the day, and got duplicates over the same day (orders get more information the day they're received, at my bank). So this value should be made settable, at least. I'm worried that adding too many parameters might confuse the user here, but that's ok to have before getting automatic duplicate merge. Also, why is the threshold value in minutes, not in hours?

Sorting bits look good, thanks.

**Comment by ZeHiro** [comment]
In fact the aim of the new treshold is to avoid to declare as duplicates two similar operations (same value, same date, same labels) imported at the same time. If they are not imported at the same time, they could be real duplicates (need to be merged) as banks usually completes the label during the day. That's why i used minutes for the treshold, i assumed an import of operations wouldn't exceed a few minutes (5 in this case).

**Comment by bnjbvr** [comment]
I see. Would it work if you just used `+a.dateImport !== +b.dateImport`, instead of a threshold?

**Comment by ZeHiro** [comment]
Yes it will work. I didn't see the import date was the same for the same batch of operations. I modify

**Comment by ZeHiro** [comment]
Fixed

File: client/components/Similarity.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function findRedundantPairs(operations, duplicateThreshold) {
     // duplicateThreshold is in hours
     var threshold = duplicateThreshold * 60 * 60 * 1000;
     DEBUG('Threshold: ' + threshold);
-
+
```
Comment (bnjbvr) [inline client/components/Similarity.js]: nit: trailing space here

**Comment by bnjbvr** [comment]
Looks good, can you remove the trailing space and update the commit message, please? (it can be done with `git commit --amend`)

**Comment by ZeHiro** [comment]
Done. Any idea for a linting/strict-syntax checking tool, so that I can push better quality code?

**Comment by bnjbvr** [comment]
Thanks! There definitely are a few linters I'd like to try. For the frontend, they need to be compatible with es2015 and jsx. The hard part will be 1) finding a good set of rules 2) fix all files so that linting succeeds.

**Comment by bnjbvr** [comment]
I've merged the PR. Thanks, it works like a charm! There's one pair of false positives on my instance that doesn't show up anymore.

**Comment by ZeHiro** [comment]
It's not perfect yet, but it gets better. I'll have one false positive left.

**Pull request merged**